src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableRecords.java

```diff
@@ -13,7 +13,7 @@ public record InputUnusedLocalVariableRecords(int a, int b) {
     public InputUnusedLocalVariableRecords {
         int ab = 12;
         ab += a;
-        int var1 = 1; // violation
+        int var1 = 1; // violation, 'Unused local variable*.'
```

Comment:
https://github.com/checkstyle/checkstyle/blob/1d86c77a95685c276812549fcb9c8280cd20f80c/src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties#L66

`// violation, 'Unused local variable'`
Or put name of variable name to message, but if case is simple, no need

Reply:
done. I didn't add the variable name because each file will require manual updates. I did it using the "replace All" option in Intelliji.
I just realized that we can close "remove unnecessary ok" issue with one click. I don't what prevents us from doing this in a single PR :)

Reply:
Yes, it is easy, but somebody needs to do this.
And there nuances.